Tests/UnitTests/AdyenTheme/AdyenThemeTests.swift

```diff
@@ -95,5 +95,174 @@ final class AdyenThemeTests: XCTestCase {
         // Then
         XCTAssertEqual(updatedTheme.toggleStyle.cornerRadius, cornerRadius)
     }
-
+    
+    func test_textfieldMethod_shouldUpdateBackgroundColor() {
```

Comment:
This file grows, please consider creating separate set of tests placed in their own file for each semantic theme element (toggle, textfield, etc)